runelite-client/src/main/java/net/runelite/client/plugins/cluescrolls/clues/MusicClue.java

```diff
@@ -43,9 +43,9 @@
 
 @AllArgsConstructor(access = AccessLevel.PRIVATE)
 @Getter
-public class MusicClue extends ClueScroll implements NpcClueScroll
+public class MusicClue extends ClueScroll implements NpcClueScroll, LocationClueScroll
 {
-	private static final WorldPoint LOCATION = new WorldPoint(2990, 3384, 0);
+	private final WorldPoint location = new WorldPoint(2990, 3384, 0);
```

Comment:
Why is `static` removed from this constant definition?

Reply:
I just followed how CoordinateClue.java implemented LocationClueScroll.

Reply:
I'd recommend overriding `getLocation()` as:

```java
@Override
public WorldPoint getLocation()
{
	return LOCATION;
}
```

rather than renaming the constant and relying on the lombok-generated getter to implement it.